CONTRIBUTING.md

```diff
@@ -0,0 +1,211 @@
+<!-- omit in toc -->
+# Contributing to AutoAgora agents
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+>
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
```

Comment:
Maybe create a [CITATION.bib](https://github.com/semiotic-ai/TheGraphOpt.jl/blob/main/CITATION.bib) for the project?

Reply:
Isn't that a bit overkill?

Reply:
I'd have to create a GH action that automatically updates it though :/